shared/sim_shared.core

```diff
@@ -10,7 +10,7 @@ filesets:
       - lowrisc:prim:assert
     files:
       - ./rtl/prim_clock_gating.sv
-      - ./rtl/ram_1p.sv
+      - ./rtl/ram_2p.sv
```

Comment:
This change is breaking CI. You have to leave the `ram_1p.sv` in there as well. This is still used for the compliance tests.

Reply:
Done. Thanks